lib/src/services/databases/dataRefresh/implementation.dart

```diff
@@ -0,0 +1,47 @@
+import "package:ramaz/models.dart";
+import "package:ramaz/services.dart";
+import "../../firestore.dart";
+import "interface.dart";
+///
+class CloudDataRefresh implements DataRefreshInterface{
+  ///
+  static final CollectionReference<Map> data = Firestore
```

Comment:
```suggestion
  static final CollectionReference<Map> collection = Firestore
```

This variable doesn't hold any data, just a reference to the collection that holds the data. See comment below as well.